src/main/java/com/template/Exception/MeteorologicalInfoNotFound.java

```diff
@@ -0,0 +1,7 @@
+package com.template.Exception;
+
+public class MeteorologicalInfoNotFound extends RuntimeException{
```

Comment:
super legal ter trazido esses custom Exceptions... um outro momento conversamos mais sobre e se isso é uma boa prática

Reply:
precisamos tb entender se o RuntimeException é o melhor para esse cenário

Reply:
> precisamos tb entender se o RuntimeException é o melhor para esse cenário

Opa Riguel, muito obrigado pela review.  Acredito que o melhor tipo nesse caso seria uma FileNotFoundException, correto? A grande questão é que eu não cosegui tratar esse tipo de exception no meu handler, o handler é esse: 
![image](https://github.com/DB-WeatherApp/dbcamp-template-api/assets/54593758/620e1877-180d-4235-a568-da4ca3baea4c)

O que eu fiz foi, extender a runtime exception, tratar ela para retornar para o usuário um HttpStatus 404 Not Found e dentro da minha service eu retorno uma mensagem personalizada, que nesse caso seria: 
![image](https://github.com/DB-WeatherApp/dbcamp-template-api/assets/54593758/8d30ae40-732b-46f1-a276-2e1586bddd8e)
No Insomia fica dessa forma: 
![image](https://github.com/DB-WeatherApp/dbcamp-template-api/assets/54593758/1200d7fe-6e76-40a9-834a-64187d69cbec)